config/config.go

```diff
@@ -67,8 +65,9 @@ func init() {
 // Load will load the necessary config
 func Load(logger logrus.FieldLogger) (Options, error) {
 	if err := viper.ReadInConfig(); err != nil {
-		logger.Warn("unable to find config.yml. loading config from environment variables")
+		errors.Wrap(err, "unable to find config.yml. loading config from environment variables")
```

Comment:
I meant `logger.Warn(errors.Wrapf(err, "").Error())`

Reply:
Oh... my bad. updating that now.